.github/templates/linux_ci_workflow.yml.j2

```diff
@@ -289,7 +289,7 @@ jobs:
         run: |
           sudo df -H
       !{{ common.parse_ref() }}
-      - name: Test
+      - name: Test ${{ github.workflow }} / test (${{ matrix.config }}, ${{ matrix.shard }}, ${{ matrix.num_shards }},  ${{ matrix.runner }})
```

Comment:
is the matrix name different from `github.job`? https://docs.github.com/en/actions/learn-github-actions/contexts#github-context

Reply:
Hmm, tried it https://github.com/pytorch/pytorch/pull/65501/commits/383fc9f3b5051f20961f0145392d8a5d1451ee00, but does not seem to yield desired results. See this run for example: https://github.com/pytorch/pytorch/pull/65501/checks?check_run_id=3705690947